datastore/CHANGELOG.md

```diff
@@ -4,6 +4,25 @@
 
 [1]: https://pypi.org/project/google-cloud-datastore/#history
 
+## 1.7.0
+
+### Implementation Changes
+
+- Do not pass 'offset' once the query iterator has a cursor (#5503)
+
+### Documentation
+
+- minor fix to datastore example (#5452)
+- Add example showing explicit unicode for text values in entities. (#5263)
+
+### Internal / Testing Changes
+
+- Add Test runs for Python 3.7 and remove 3.4 (#5295)
```

Comment:
which section do you think it should go in? As far as I know, we didn't outright drop support for 3.4, we just stopped testing it.

Reply:
> Which section do you think it should go in?

Dunno -- I don't have a feel for those classifications.  E.g., I would highlight bugfixes in their own section, separate from refactorings / test changes, precisely because they are / should be user-visible.

>  As far as I know, we didn't outright drop support for 3.4, we just stopped testing it.

ISTM if we quit testing it, then we should drop the Trove classifier, too.

Reply:
Moved it into implementation changes.

Reply:
Should we remove the classifier in `setup.py` as well?